contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -573,14 +715,46 @@ contract NodeOperatorsRegistry is INodeOperatorsRegistry, AragonApp, IStakingMod
         usedSigningKeys = nodeOperator.depositedSigningKeysCount;
     }
 
+    function getNodeOperatorLimits(uint256 _nodeOperatorId)
```

Comment:
Can we have an additional method that will return all limits for all node operators or in a bunch of `from` - `to`?